outline.md

```diff
@@ -0,0 +1,62 @@
+* Introduction
+  - A Brief History of Open
```

Comment:
Is this a typo? or it is meant to be like this? Intuitively I would expect "Open something"

Reply:
It's not a typo, but the fact that you hesitated on reading it indicates that maybe we should change the wording.

For now I've changed it to "A Brief History of Open Licensing".  I don't like focusing on licensing like that (as opposed to focusing on collaboration), but we're not presenting a brief history of open collaboration throughout history -- that would go back many thousands of years! -- but rather a brief history of the digital collaboration that arose from open licensing.

We may think of something better, and suggestions welcome!  This isn't written in stone, naturally.

Reply:
I would suggest for the scope of the toolkit to narrow this down to a brief history of open licensing in the public sector specifically.